Update documentation to v1.0.0.

DWesl opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->
Hopefully this fixes #42.

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->
This brings the published version of the documentation up-to-date with the code version, which is useful given the recent release.

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->
I checked locally to make sure MathJax renders

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [X] I give permission for my code to be distributed under the existing license
<!-- this may need to be okayed by your company's legal department if you did this on work time. -->

DWesl committed:
Update documentation to v1.0.0.

Hopefully this fixes #42.

zmoon commented on _modules/atmos_flux_inversion.html:
```diff
@@ -15,6 +15,8 @@
     <script type="text/javascript" src="../_static/underscore.js"></script>
     <script type="text/javascript" src="../_static/doctools.js"></script>
     <script type="text/javascript" src="../_static/language_data.js"></script>
+    <script async="async" type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-AMS-MML_HTMLorMML"></script>
+    <script type="text/x-mathjax-config">MathJax.Hub.Config({"extensions": ["tex2jax.js"], "jax": ["input/TeX", "output/HTML-CSS"], "TeX": {"extensions": ["AMSmath.js"]}})</script>
```
example where MathJax config is specified twice in the HTML output

DWesl replied:
I see where the `MathJax.Hub.Config(...)` line comes from, but I don't know where the `config=TeX-AMS-MML_HTMLorMML` part comes from.  I'm suspecting that's added by `sphinx.ext..mathjax` itself.  I'll look into how to change that.

zmoon replied:
yeah it seems like that might be the default `mathjax_path` setting
https://github.com/sphinx-doc/sphinx/blob/a721631a541db298f7f9807c9db8433e928cb20b/sphinx/ext/mathjax.py#L98

zmoon replied:
so I guess workaround could be to set `mathjax_path` in your conf to `'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js'`

zmoon replied:
or remove your `mathjax_config`, or just not worry about it

DWesl replied:
Updating to latest Sphinx leaves the duplicate configuration.  I dropped the manual MathJax configuration, which leaves just the first line and seems to display the equations.

DWesl committed:
Update sphinx version.

This may allow reduce duplicated MathJax configuration, but does not
appear to do so.  I'm not sure what the warning in the docs is for.

DWesl committed:
Remove the duplicate MathJax configuration.

MathJax 3 changes how configuration works, and Sphinx 4 updates to
MathJax 3.  This should help avoid problems when I update Sphinx.
